Makefile

```diff
@@ -140,12 +141,19 @@ test-e2e: ## Run end-to-end tests using an available cluster.
 ##@ Release Artifacts
 ######################
 
-cross-platform: darwin linux windows ## Build all distributable (cross-platform) binaries
+cross-platform: darwin-arm64 darwin-amd64 linux windows ## Build all distributable (cross-platform) binaries
 
-darwin: $(BIN_DARWIN) ## Build for Darwin (macOS)
+darwin-arm64: $(BIN_DARWIN_ARM64) ## Build for mac M1
+	env CGO_ENABLED=0 GOOS=darwin GOARCH=arm64 go build -o $(BIN_DARWIN_ARM64) -ldflags $(LDFLAGS) ./cmd/$(BIN)
 
-$(BIN_DARWIN): zz_filesystem_generated.go
-	env CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 go build -o $(BIN_DARWIN) -ldflags $(LDFLAGS) ./cmd/$(BIN)
+$(BIN_DARWIN_ARM64): zz_filesystem_generated.go
+	env CGO_ENABLED=0 GOOS=darwin GOARCH=arm64 go build -o $(BIN_DARWIN_ARM64) -ldflags $(LDFLAGS) ./cmd/$(BIN)
+
+darwin-amd64: $(BIN_DARWIN_AMD64) ## Build for Darwin (macOS)
```

Comment:
why is this different that the other two?

Reply:
nevermind, my bad

Reply:
Just: I find `darwin-arm64` and `darwin-arm64` as duplicates of `$(BIN_DARWIN_ARM64)` and `(BIN_DARWIN_AMD64)`.
Am I missing something @lance

Reply:
Looks like `windows` and `linux` recipes are basically aliases -- there is only prerequisite and no commands.
The same is not trues for `darwin-arm64` and `darwin-arm64`.

Reply:
@lance ^^^

Reply:
Oh I see now. I was just following the existing pattern for the previous `darwin` target. But it seems like that was duplicative.

Reply:
@lkingland what's correct here?

Reply:
That's right, these are simple aliases to follow the pattern of a phony target which will always build the targeted binary even if the file already exists.  (and IIRC .PHONY is assumed in modern versions of make).  It also gives a nice attachment point for the `make help` text. So this PR looks basically right, it just needs the _contents_ of the alias command removed as Matej points out.